### PR TITLE
codewhisperer: fix unintended change which is causing server side error "Exceed supplemental context content length"

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## Problem
revert unintended change introduced with this PR
https://github.com/aws/aws-toolkit-vscode/pull/3737/files#diff-d20df5e39cab67435d8923babae2de0a72f4e946e91f24111bf694bf28bb5a8eL73

This misconfiguration will change number of lines in a code chunk from 10 to 60 & 1000.
1000 lines of code in a code chunk will make the plugin to fetch the entire file and send back to the server every time, and we have a limitation of 10240 char length on supplemental context, thus users are possibly getting validation error from the service.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by justinmk3** [comment]
seems like this is worth mentioning as a fix in the changelog

**Review by justinmk3 — APPROVED** [review]

**Comment by Will-ShaoHua** [comment]
> 

yea @justinmk3 I will add a changelog for this

File: .changes/next-release/Feature-c4e5cdd8-cc0e-4006-b06a-91e616ffc251.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
```
Comment (Will-ShaoHua) [inline .changes/next-release/Feature-c4e5cdd8-cc0e-4006-b06a-91e616ffc251.json:2]: will fix this

**Review by andrewyuq — APPROVED** [review]

**Comment by Will-ShaoHua** [comment]
@justinmk3 CI is failing on `SamCliLocalInvokeInvocation`
```
  1) SamCliLocalInvokeInvocation
       invokes `sam local` with args:
     Error: Test length exceeded max duration: 30 seconds
```

should we re-run the CI?

File: .changes/next-release/Bug Fix-880d1db0-1a68-4fbb-96c8-af1a988238bb.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "CodeWhisperer: fix error exceed max supplemental context length"
```
Comment (justinmk3) [inline .changes/next-release/Bug Fix-880d1db0-1a68-4fbb-96c8-af1a988238bb.json:3]: How does the customer perceive the issue? Will this description mean anything to a customer? 

If the impact is something like "CodeWhisperer sometimes fails on big files", then that is what the changelog item should say.
```suggestion
	"description": "CodeWhisperer sometimes fails on big files"
```

Comment (Will-ShaoHua) [reply .changes/next-release/Bug Fix-880d1db0-1a68-4fbb-96c8-af1a988238bb.json]: sgtm

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**